src/routes/sessions/[id]/+page.svelte

```diff
@@ -109,6 +105,29 @@
 		await handleCompletion(payload);
 	}
 
+	async function handleSubmitEditMessage() {
+		if (messageIndexToEdit === null) return;
+
+		session.messages[messageIndexToEdit].content = prompt;
+
+		// Remove all messages after the edited message
+		session.messages = session.messages.slice(0, messageIndexToEdit + 1);
+
+		let payload = { model: session.model, messages: session.messages, stream: true };
+		await handleCompletion(payload);
```

Comment:
Let's get rid of the single use of `payload` and move the props directly inside the `handleCompletion()`:
```suggestion
		await handleCompletion({ model: session.model, messages: session.messages });
```
Because we used a the `payload` variable we didn't notice that `stream: true` wasn't actually needed.

There are other similar single-uses of `payload` in `handleRetry(payload)` and `handleSubmitNewMessage(payload)`, let's update those as well.

Reply:
Note: Now that I've removed the `stream: true`, one of the tests is failing. I don't exactly see why, because when I run `npm run test` on my computer it works fine.

Reply:
It's weird that it's working in your computer, but the test is failing because we it's expecting `stream: true` to be there but we just removed it, so it's good thing that the test caught it 👍 

![image](https://github.com/user-attachments/assets/6d7109ae-d5cb-4996-9143-7bfcab7a0720)

In this case I think it's safe to remove it because Ollama's `stream` parameter defaults to `true`.
But just to double check make do some QA to make sure Ollama is indeed streaming a response back.

Reply:
Thank you! I've removed it from the test and it's working now.